install/resources/upgrade50.php

```diff
@@ -35,3 +35,22 @@ function upgrade50_dbchanges()
 	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
 	$output->print_footer("50_done");
 }
+
+function upgrade50_verify_adminemail()
+{
+	global $output, $cache, $db, $mybb;
+
+	$output->print_header("Verifying Admin Email");
+	if(empty($mybb->settings['adminemail']))
+	{
+		echo "<p>Updating admin email settings...</p>";
+		echo "<p><small>P.D: Field can not be empty</small></p>";
+		$db->update_query('settings', array('value' => $mybb->user['email']), "name='adminemail'");
```

Comment:
Maybe you can add a simple input field for users to insert an email here? ... ->
```suggestion
		if(!isset($mybb->input['adminemail']))
		{
			$mybb->input['adminemail'] = $mybb->user['email'];
		}
		echo "<input type=\"text\" class=\"text_input\" name=\"adminemail\" id=\"adminemail\" value=\"".htmlspecialchars_uni($mybb->input['adminemail'])."\" />";
```